### DISCUSSION TITLE
DuckDB Connector

### DISCUSSION THREAD

**Discussion by kentstephen** [dekart-xyz/dekart#186] [Ideas]
Hi, 

Someone sent me a Kepler map via Dekart today - it's very cool. 

He used Big Query for the map, but it occurred to me that the S3 bucket could be queried directly using DuckDB with the
```httpfs``` extension. 

A lot of open geo data can be queried with DuckDB. Any interest in supporting? I feel like DuckDB was made for something like this. 

I saw Postgres on the site, maybe I missed it. 

Thanks for your time.

**Comment by delfrrr** [comment]
Hi @kentstephen , DuckDB is at back of my mind, you are totally right. My goal with Dekart is building ultimate tools for creating and publishing keeper.gl maps. DuckDB is popular choice for preparing the data for the map.

Would you consider contribution? I'm happy to jump on call to brainstorm, let me know.

**Comment by kentstephen** [comment]
I messaged you on LinkedIn. I am definitely interested in being involved,
let me know - I’m happy to chat.

**Comment by kentstephen** [comment]
This is a somewhat similar project I’ve used with DuckDB

https://github.com/Youssef-Harby/OvertureMapsDownloader
